prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm_states.c

```diff
@@ -89,7 +89,21 @@ static enum fi_rdm_tagged_req_hndl_ret
 fi_ibv_rdm_tagged_rndv_recv_ack_lc(struct fi_ibv_rdm_tagged_request *request,
 		void *data);
 static enum fi_rdm_tagged_req_hndl_ret
-fi_ibv_rdm_tagged_init_rma_request(struct fi_ibv_rdm_tagged_request *request,
+fi_ibv_rdm_rma_init_request(struct fi_ibv_rdm_tagged_request *request,
+		void *data);
+static enum fi_rdm_tagged_req_hndl_ret
+fi_ibv_rdm_rma_inject_request(struct fi_ibv_rdm_tagged_request *request,
+		void *data);
+static enum fi_rdm_tagged_req_hndl_ret
+fi_ibv_rdm_rma_post_ready(struct fi_ibv_rdm_tagged_request *request,
+		void *data);
+static enum fi_rdm_tagged_req_hndl_ret
+fi_ibv_rdm_rma_inject_lc(struct fi_ibv_rdm_tagged_request *request, void *data);
+static enum fi_rdm_tagged_req_hndl_ret
+fi_ibv_rdm_rma_buffered_lc(struct fi_ibv_rdm_tagged_request *request,
+		void *data);
+static enum fi_rdm_tagged_req_hndl_ret
+fi_ibv_rdm_rma_zerocopy_lc(struct fi_ibv_rdm_tagged_request *request,
```

Comment:
Is it possible to restructure the code to avoid needing these forward declarations?